staging/src/k8s.io/client-go/util/certificate/csr/csr.go

```diff
@@ -62,6 +62,9 @@ func RequestCertificate(client clientset.Interface, csrData []byte, name string,
 	if len(csr.Name) == 0 {
 		csr.GenerateName = "csr-"
 	}
+	if duration != 0 {
```

Comment:
Should we attempt to match apiserver validation? i.e. >= min?

Reply:
You get a nice validation error from the server if you pass in something that does not make sense?